app/controllers/comments_controller.rb

```diff
@@ -64,12 +69,12 @@ def set_comment
       @comment = Comment.find(params[:id])
     end
 
-    def is_an_authorized_user
-      @photo = Photo.find(params.fetch(:comment).fetch(:photo_id))
-      if current_user != @photo.owner && @photo.owner.private? && !current_user.leaders.include?(@photo.owner)
-        redirect_back fallback_location: root_url, alert: "Not authorized"
-      end
-    end
+    # def is_an_authorized_user
+    #   @photo = Photo.find(params.fetch(:comment).fetch(:photo_id))
+    #   if current_user != @photo.owner && @photo.owner.private? && !current_user.leaders.include?(@photo.owner)
+    #     redirect_back fallback_location: root_url, alert: "Not authorized"
+    #   end
+    # end
 
```

Comment:
Try not to leave commented out code unless there is a note describing the purpose of the comment.